js/main.js

```diff
@@ -26,4 +28,102 @@ $form.addEventListener('submit', (e) => {
   $imgSrcChange.setAttribute('src', './images/placeholder-image-square.jpg');
 
   $form.reset();
+
+  // render the renderEntry callback on the form if the form's submitted
+
+  const callRender = renderEntry(objStorageFormInputs);
```

Comment:
Needs a `$` in front of the variable name. This is technically storing a DOM element